subsys/bluetooth/controller/ll_sw/ull_iso_internal.h

```diff
@@ -4,19 +4,20 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-#if defined(CONFIG_BT_CTLR_ADV_ISO) || defined(CONFIG_BT_CTLR_CONN_ISO)
 /* Calculate ISO PDU buffers required considering SDU fragmentation */
-/* FIXME: Calculation considering both Connected and Broadcast ISO PDU
- *        fragmentation.
- */
-#if defined(CONFIG_BT_CTLR_CONN_ISO)
-#define BT_CTLR_ISO_TX_BUFFERS (((CONFIG_BT_CTLR_CONN_ISO_SDU_LEN_MAX + \
-				  CONFIG_BT_CTLR_CONN_ISO_PDU_LEN_MAX - 1U) / \
-				 CONFIG_BT_CTLR_CONN_ISO_PDU_LEN_MAX) * \
+#if defined(CONFIG_BT_CTLR_ADV_ISO) && defined(CONFIG_BT_CTLR_CONN_ISO)
+#define BT_CTLR_ISO_TX_BUFFERS (DIV_ROUND_UP((CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE - 4U), \
+					     MIN(CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX, \
+						 CONFIG_BT_CTLR_CONN_ISO_PDU_LEN_MAX)) * \
+				CONFIG_BT_CTLR_ISO_TX_BUFFERS)
+#elif defined(CONFIG_BT_CTLR_ADV_ISO)
+#define BT_CTLR_ISO_TX_BUFFERS (DIV_ROUND_UP((CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE - 4U), \
+					     CONFIG_BT_CTLR_ADV_ISO_PDU_LEN_MAX) * \
+				CONFIG_BT_CTLR_ISO_TX_BUFFERS)
+#elif defined(CONFIG_BT_CTLR_CONN_ISO)
+#define BT_CTLR_ISO_TX_BUFFERS (DIV_ROUND_UP((CONFIG_BT_CTLR_ISO_TX_BUFFER_SIZE - 4U), \
+					     CONFIG_BT_CTLR_CONN_ISO_PDU_LEN_MAX) * \
```

Comment:
If `4` here refers to `BT_HCI_ISO_SDU_HDR_SIZE`, then please use that. 

If it refers to `BT_HCI_ISO_HDR_SIZE` then please use that

I believe it's for the latter, right? As the SDU header is not part of the ISO TX buffer size